site/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     },
     "references": [
         { "path": "../clientUtils" },
+        { "path": "../db" },
```

Comment:
Is still still necessary? I didn't notice any imports from `db`

Reply:
nope, thanks!